test/integration/storageversionmigrator/util.go

```diff
@@ -829,12 +830,37 @@ func (svm *svmTest) createCR(ctx context.Context, t *testing.T, crName, version
 		},
 	}
 
-	crdUnstructured, err := svm.dynamicClient.Resource(crdResource).Namespace(defaultNamespace).Create(ctx, crdUnstructured, metav1.CreateOptions{})
+	var lastError error
+	err := wait.ExponentialBackoff(wait.Backoff{
```

Comment:
Do we really need a custom backoff here?

Reply:
In all the recent flakes I see following error 
```shell
util.go:1042: Failed to create CR: client rate limiter Wait returned an error: context deadline exceeded
```

Reply:
All the error says is that the context got canceled meaning it timed out.  Retrying won't help because the context is already cancelled so all retries will fail.